apps/remix-ide/src/app/tabs/analysis-tab.js

```diff
@@ -47,8 +57,18 @@ class AnalysisTab extends ViewPlugin {
     }
 
     this.event.register('staticAnaysisWarning', (count) => {
+      let payloadType = ''
+
+      this.hints && this.hints.length > 0 ? this.hints.forEach(hint => {
```

Comment:
you could do a normal for/loop and break it as soon as you encounter an  error type, then you don't need to loop over the entire set.